src/FlashlessScript.tsx

```diff
@@ -11,9 +13,6 @@ const baseVariables = {
   '--badge-text': ['white', 'whiteAlpha.800']
 };
 
-type Color = string | [string, number];
-type Variables = Record<string, [Color, Color]>;
```

Comment:
These lines could just say `export type` and we wouldn't need to break them out into their own file, if you're cool with the ergonomics of importing the types as top-level named exports of the package.

```ts
import { Color, Variables } from 'chakra-ui-flashless';
```